packages/app/src/specs/generators/component/ReactComponentList.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <div class="h-full">
+    <div
+      v-if="!loading && !components.length"
+      class="border-b-1 border-b-gray-50 py-2 pl-56px"
+    >
+      {{ t('createSpec.noComponentsFound') }}
+    </div>
+    <ul v-else>
+      <li
+        v-for="{exportName, isDefault} in components"
+        :key="exportName"
+      >
+        <div
+          class="cursor-pointer flex border-b-1 border-b-gray-50 leading-normal pl-56px text-16px
+    gap-8px group items-center last last:py-0 last:items-start children:h-40px children:py-8px"
+          @click="$emit('selectItem', {file, item: {exportName, isDefault}})"
+        >
+          <div
+            class="h-full inline-flex whitespace-nowrap items-center overflow-hidden"
+          >
+            <i-cy-puzzle-piece_x16
```

Comment:
I feel like this could use a hover effect. The Files have a noticeable effect on hover (dropdown icon color change, relative path) but then it feels like goes stale hovering over these elements

Reply:
good catch. i missed this when we switched directions. i would use the pattern we have for a runner even though a file here is not a folder it acts as a container.
![expanded](https://user-images.githubusercontent.com/1778311/206550565-08ccaae2-4ad2-47fc-a960-35c6b878cd29.png)
![list](https://user-images.githubusercontent.com/1778311/206550573-d7807c3c-0a0c-4dec-827f-2379e19afeb8.png)

Reply:
Added a hover effect. You can also now see it in the mocks that are linked in the PR description